core/chains/evm/gas/block_history_estimator.go

```diff
@@ -976,3 +974,29 @@ func (b *BlockHistoryEstimator) EffectiveTipCap(block evmtypes.Block, tx evmtype
 		return nil
 	}
 }
+
+// Int64ToHex converts an int64 into go-ethereum's hex representation
+func Int64ToHex(n int64) string {
+	return hexutil.EncodeBig(big.NewInt(n))
+}
+
+// HexToInt64 performs the inverse of Int64ToHex
+// Returns 0 on invalid input
+func HexToInt64(input interface{}) int64 {
```

Comment:
Where is this used?

Reply:
[Here](https://github.com/smartcontractkit/chainlink/blob/develop/core/chains/evm/gas/block_history_estimator.go#L700) and a few other places in BHE. I just moved them, nothing new was added.

Reply:
Do they need to be exported?